waves/src/theme.tsx

```diff
@@ -17,44 +17,21 @@ const theme = extendTheme({
     },
     components: {
         Button: {
-            baseStyle: {
-                bg: "#304FFE",
-                fontWeight: "bold",
-            },
-
-            sizes: {
-                lg: {
-                    fontSize: "lg",
-                },
-            },
-            // Custom variant
             variants: {
-                "tx_button": {
-                    bg: "blue.100",
-                    color: "white",
-                },
-                "main_button": {
-                    w: "15rem",
+                "primary": {
                     color: "white",
+                    bg: "blue.500",
                     _hover: {
                         bg: "blue.300",
                         _disabled: {
-                            bg: "",
+                            bg: "blue.300",
```

Comment:
Did you try if this works as expected? 

If the button is `disabled` it has a different color (usually brighter) and we used to have a weird bug here where hovering over a disabled button caused the button to disappear (turn almost completely white).

Reply:
|Disabled|Hover & Disabled|
|---|---|
|![image](https://user-images.githubusercontent.com/5486389/106995916-0801de80-67d4-11eb-8c71-7706be2dbd68.png)|![image](https://user-images.githubusercontent.com/5486389/106995939-105a1980-67d4-11eb-8447-0b0db95763d1.png)|

Reply:
cool 👍